source/MRViewer/MRSpaceMouseHandlerWindows.h

```diff
@@ -15,6 +15,11 @@ class SpaceMouseHandlerWindows : public SpaceMouseHandler, public MultiListener<
     virtual void initialize() override;
     virtual void handle() override;
     virtual void updateConnected( int jid, int event );
+
+    // set mode disabling zoom by mouse scroll (to fix scroll signal from spacemouse driver)
+    MRVIEWER_API void setDisableMouseScrollZoom( bool disableMouseSrollZoom );
+    // get mode disabling zoom by mouse scroll
+    bool getDisableMouseScrollZoom() { return disableMouseScrollZoom_; }
```

Comment:
`setDisableMouseScrollZoom -> activateMouseScrollZoom`
`getDisableMouseScrollZoom -> isMouseScrollZoomActive`

not to use negative statement in function names

Reply:
fixed